From 82f0c2041d62c6c1fe220d4aa7e1663dac3262f1 Mon Sep 17 00:00:00 2001
From: Chuanhong Guo <gch981213@gmail.com>
Date: Fri, 21 Jun 2019 18:33:53 +0800
Subject: [PATCH 1/2] ath79: add wmac migration for all ar93xx/qca95xx SoCs

Add migration for all ar71xx device path as well as previously
incorrect ath79 path.

Signed-off-by: Chuanhong Guo <gch981213@gmail.com>
---
 .../etc/hotplug.d/ieee80211/00-wifi-migration         | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/target/linux/ath79/base-files/etc/hotplug.d/ieee80211/00-wifi-migration b/target/linux/ath79/base-files/etc/hotplug.d/ieee80211/00-wifi-migration
index b1824303e6..37a1346fe7 100644
--- a/target/linux/ath79/base-files/etc/hotplug.d/ieee80211/00-wifi-migration
+++ b/target/linux/ath79/base-files/etc/hotplug.d/ieee80211/00-wifi-migration
@@ -24,11 +24,12 @@ migrate_wifi_path() {
 				;;
 			esac
 		;;
-		"platform/qca955x_wmac")
-			path="platform/ahb/ahb:apb/18100000.wmac"
-			WIFI_PATH_CHANGED=1
-		;;
-		"platform/ar933x_wmac")
+		"platform/ahb/ahb:apb/18100000.wmac"|\
+		"platform/ar933x_wmac"|\
+		"platform/ar934x_wmac"|\
+		"platform/qca953x_wmac"|\
+		"platform/qca955x_wmac"|\
+		"platform/qca956x_wmac")
 			path="platform/ahb/18100000.wmac"
 			WIFI_PATH_CHANGED=1
 		;;
-- 
2.20.1.windows.1

